### PR TITLE
Feat: Add Mission Section to Webpage

### LINKED ISSUE #3: Develop Missions section
Develop a pixel-perfect missions section as discussed

Figma link:
https://www.figma.com/file/BTqsUJY9pGEmLqLoIHt84L/DevRel-Community-Africa?type=design&node-id=0%3A1&mode=design&t=gtN69upHhKOj4695-1

### PR DESCRIPTION
**Pull request by DesmondSanctity**
- [x] Mission Section
- [x] Responsiveness
- [x] Add Vector

Fixes #3

### CODE REVIEW

**Comment by hacktivist123** [comment]
Hello @DesmondSanctity is this ready for merge? 

What is left?

**Comment by DesmondSanctity** [comment]
> Hello @DesmondSanctity is this ready for merge? 
> 
> What is left?

Not yet ready. There's a vector image I need to add in that mission section background. I will work on it this weekend.

**Comment by hacktivist123** [comment]
> > Hello @DesmondSanctity is this ready for merge?
> > What is left?
> 
> Not yet ready. There's a vector image I need to add in that mission section background. I will work on it this weekend.

You sure? I could just work on it myself tonight

**Comment by DesmondSanctity** [comment]
> > > Hello @DesmondSanctity is this ready for merge?
> > > What is left?
> > 
> > Not yet ready. There's a vector image I need to add in that mission section background. I will work on it this weekend.
> 
> You sure? I could just work on it myself tonight 

Yeah definitely. I will do it

**Comment by codejagaban** [comment]
Hi @DesmondSanctity What's the update on this issue?

File: components/organisms/MissionSection/MissionSection.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import Image from "next/image";
+
+export function MissionSection() {
+  const missions = [
+    {
+      id: 1,
+      title:
+        "Educate more developers and technical writers in Africa about the significance of developer advocacy and how to pursue a career in this field.",
+      titleS:
```
Comment (codejagaban) [inline components/organisms/MissionSection/MissionSection.tsx:10]: What's this `titleS` exactly?

File: components/organisms/MissionSection/MissionSection.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    },
+  ];
+
+  return (
+    <section className="bg-green-100 py-10">
+      <div className="container mx-auto px-2 py-2">
+        <div className="text-neutral-800 text-[32px] font-semibold">
+          The Mission
+        </div>
+        <div className="mt-6 sm:mt-6 md:mt-8 lg:mt-10 text-neutral-600 text-sm font-[300] sm:text-lg md:text-xl lg:text-2xl font-normal">
```
Comment (codejagaban) [inline components/organisms/MissionSection/MissionSection.tsx]: Why do you have `font-[300]` and `font-normal` in the same place?

File: components/organisms/MissionSection/MissionSection.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          promote the field of developer relations in the region.
+        </div>
+      </div>
+      <div className="container mx-auto px-2 py-2">
+        <div className=" mt-6 grid grid-cols-2 gap-2 md:grid-cols-2 lg:grid-cols-3 gap-6">
+          {/* Card 1 */}
+          {missions &&
+            missions.map((mission) => (
+              <div
+                className="bg-[#FFF] p-2 h-[120px] sm:h-[300px] md:h-[250px] lg:h-[300px] xl:h-[280px]"
```
Comment (codejagaban) [inline components/organisms/MissionSection/MissionSection.tsx]: why can't you use `bg-white` here? Or is white a custom colour?🤔
Also are reason why you're specifying the height of of this div for every screen size?🤔

IMO I don't think it's necessary, also the height doesn't match what is on the designs on Figma

File: components/organisms/MissionSection/MissionSection.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+                key={mission.id}
+              >
+                <div className=" mt-2 mb-6 w-12 h-px left-0 top-0 border border-gray-400"></div>
+                <div className="hidden sm:block w-85 left-0 top-[16px] text-emerald-700 text-[10px] font-[8] sm:text-xl md:text-xl lg:text-2xl font-normal">
+                  {mission.title}
+                </div>
+                <div className="block sm:hidden w-85 left-0 top-[16px] text-emerald-700 text-[10px] font-[8] sm:text-xl md:text-xl lg:text-2xl font-normal">
+                  {mission.titleS}
```
Comment (codejagaban) [inline components/organisms/MissionSection/MissionSection.tsx]: Why exactly do we have two `mission.title`?

File: components/organisms/MissionSection/MissionSection.tsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+                </div>
+              </div>
+            ))}
+        </div>
+        {/* <div className="mt-12 flex flex-col sm:flex-row gap-10">
```
Comment (codejagaban) [inline components/organisms/MissionSection/MissionSection.tsx]: Remove comments

File: tailwind.config.js [hunk 6]
```diff
@@ -20,6 +20,15 @@ module.exports = {
         heading: ["Epilogue", "sans-serif"],
         sans: ["Epilogue", "sans-serif"],
       },
+      gridTemplateColumns: {
+        "3-auto": "repeat(3, minmax(0, 1fr))",
+      },
+      screens: {
```
Comment (codejagaban) [inline tailwind.config.js:26]: This comes by default with tailwindCSS why do we need to specify them here?

**Review by codejagaban — CHANGES_REQUESTED** [review]

**Pull request merged**